### DISCUSSION TITLE
Playback Rate setting

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#8752] [Ask A Question]
In version 12 while playing back a clip you could just click on the "Playback Rate" and for each click the rate would increment to the next playback speed. With version 13 Beta you have to expressely select the playback rate.
I like the version 12 behaviour. Is there a setting in the yaml file to select this function behaviour?
Thank you

**Comment by NickM-27** [comment]
I never saw this behavior, the playback rate always had to be selected from the dropdown

**Reply by NickM-27** [reply]
it appears as though it always worked this way in chrome but other browsers worked differently previously. That said I don't think this will change in 0.13

**Reply by 18923** [reply]
I'll try and paste a Video Capture using  Chrome. BTW my iPhone behaves the same using the Home Assistant App..
Note: the mouse movement on the video capture below did not capture correctly, I only left click on play back rate and the rate changes and loops.

BTW, I just tried Edge with the same results.
Not a big deal, just tought I let you know.

https://github.com/blakeblackshear/frigate/assets/59022856/0748d36d-fd47-4bd0-96c6-c88cad4b06e6
